### PR TITLE
What's New 97

### PR DESCRIPTION
**Pull request by mikehoffms**
**Rendered page for review:**
https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/whats-new/2022/01/devtools?branch=pr-en-us-1664

### CODE REVIEW

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/01/devtools.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+## 3D View tool supports changing color themes in DevTools
+
+<!-- Title: 3D View better integrates with different themes in DevTools -->
+<!-- Subtitle: The 3D View tool now works when you select a different color theme in DevTools. -->
+
+In previous versions of Microsoft Edge, changing the theme in DevTools and then opening the 3D View tool resulted in a blank panel.  This issue has now been fixed, in Microsoft Edge 97.  To learn more, see [Apply color themes to DevTools](../../../customize/theme.md) and [3D View](../../../3d-view/index.md).
+
+![The 3D View tool now supports changing color themes.](../../media/2022/01/3d-view-with-color-theme.png)
+
+For more information about the issue that was fixed, see [Bug 36842109: 3D View panel blank after DevTools reload](https://dev.azure.com/microsoft/Edge/_workitems/edit/36842109/) and [Pull request 6615892: Fix for blank screen in 3D View after tools reloaded](https://dev.azure.com/microsoft/Edge/_git/chromium.devtools-frontend/pullrequest/6615892).
```
Comment (zoherghadyali) [inline microsoft-edge/devtools-guide-chromium/whats-new/2022/01/devtools.md:82]:
```suggestion
```

These are ADO links and would not work for external readers. I maybe meant to leave them as a comment in the ADO work item if you needed extra info for the announcement. This entire paragraph should be removed.

**Review by zoherghadyali — APPROVED** [review]

**Comment by hxlnt** [comment]
Focus Mode section: No changes, LGTM!

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/01/devtools.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+<!-- Title: Screeencast improvements and launch options for Microsoft Edge in the Visual Studio Code extension -->
+<!-- Subtitle: The correct list of emulated devices is shown for the screeencast, the correct device emulation is displayed, and there are now launch arguments for the browser. -->
+
+The current version of the Microsoft Edge DevTools extension for Visual Studio Code fixes several issues that were reported by the community:
+
+*  The list of emulated devices in the screencast is now the same as the list of emulated devices in the browser.
+
+*  Device emulation now adds the correct user agent string, to trigger the correct display.
+
+*  You now get a warning when you use a deprecated launch version of Microsoft Edge, indicating that you'd miss out on features by using a deprecated version of Microsoft Edge.
```
Comment (codepo8) [inline microsoft-edge/devtools-guide-chromium/whats-new/2022/01/devtools.md:52]: Let's remove the "by using a deprecated version of Microsoft Edge." as this is about how Edge is launched from the VS Code side, this isn't about the version of Edge. And it means we don't repeat "deprecated".

**Review by codepo8 — CHANGES_REQUESTED** [review]
One change to the VS Code extension wording.

**Pull request merged**